### ISSUE TITLE
The harvesters.dat file is not generated.

### ISSUE THREAD

**Issue by GodTanu** [nemtech/catapult-server#107]
I built a Dual node with symbol-bootstrap ver0.2.1.
symbolServerToolsImage: 'symbolplatform/symbol-server:tools-gcc-0.10.0.3'
symbolServerImage: 'symbolplatform/symbol-server:gcc-0.10.0.3'
symbolRestImage: 'symbolplatform/symbol-rest:2.1.0'
mongoImage: 'mongo:4.2.6-bionic'

I have performed a Delegated Harvesting operation and the "harvesters.dat file" is not generated.

I am aware of the following problems
https://github.com/nemtech/symbol-sdk-typescript-javascript/pull/714
https://github.com/nemtech/symbol-sdk-typescript-javascript/issues/713

I used the following tools to address this issue in the interim.
https://gist.github.com/tomotomo9696/44a4bb7ee6b0749327695605a9aa98b2
`persistentDelegationRequestTx.message.payload = persistentDelegationRequestTx.message.payload.replace(/^E201735761802AFE/i, "FE2A8061577301E2");
`

I have been successful in delegated harvesting, but there are cases where it fails. The cause is the node.

If you specify the public key for this node that Jaguar told me about, it will be successful. 
https://nem2.slack.com/archives/C9YKR0EUX/p1604925917139400?thread_ts=1604544698.119700&cid=C9YKR0EUX
`81890592F960AAEBDA7612C8917FA9C267A845D78D74D4B3651AF093E6775001`

The public keys of the other nodes will fail.
In the failed case, there is no harvesters.dat file on that node.

Furthermore, this public key is not in the node list.
Key:81890592F960AAEBDA7612C8917FA9C267A845D78D74D4B3651AF093E6775001
https://symbolnodes.org/boot_testnet/

Are there any known issues on the server side?
Do I need to configure any nodes?
Will this be improved in 0.10.0.4?
Please advise.Thanks.

**Comment by Jaguar0625** [comment]
> The public keys of the other nodes will fail.

I think you are using the wrong public key. You should be using the key stored in the `node.key.pem` file on the remote server (this is the key used in authenticating data for transport over TLS). Generally, servers that want to host delegated harvesters should publish this information. You should **NOT** be using the signing public key. 

> Are there any known issues on the server side? Will this be improved in 0.10.0.4?

No known issues on server. No planned improvements.

**Comment by GodTanu** [comment]
Thanks for the answer.
I was referring to an old document and I had the wrong configuration values.

>Generally, servers that want to host delegated harvesters should publish this information. 

Sorry, I need to know.
There is only "BEGIN PRIVATE KEY" in node.key.pem.
How do I set the public key?
How can a node publish this public key?

**Comment by Jaguar0625** [comment]
```bash
$ openssl pkey -inform PEM -text -noout -in node.key.pem
```

**Comment by fboucquez** [comment]
Is this the node public key for both 1) NodeKeyLinkTransaction and 2) PersistentDelegationRequestTransaction

**Comment by GodTanu** [comment]
I generated and specified a public key and it was successful.
I made sure that harvesters.dat was created on my node.

Both NodeKeyLinkTransaction and NodeKeyLinkTransaction are public keys generated from node.key.pem.


http://explorer-0.10.0.x-01.symboldev.network/transactions/017107D54524D3B12890FA4551C2642F8C5029590C83B23D0CF01F89AA1A5B7C

http://explorer-0.10.0.x-01.symboldev.network/transactions/03F2F93930A82DF6B13BA0F0BFE1843E0F86CF137BCBDF05BAA67F5D9CAFE02B
